orchagent/aclorch.h

```diff
@@ -242,6 +245,12 @@ class AclOrch : public Orch, public Observer
         return m_countersTable;
     }
 
+    // FIXME: Add getters for them? I'd better to add a common directory of orch objects and use it everywhere
+    PortsOrch *m_portOrch;
+    MirrorOrch *m_mirrorOrch;
+    NeighOrch *m_neighOrch;
+    RouteOrch *m_routeOrch;
```

Comment:
class AclRuleL3 inherits from AclRule, but not from AclOrch.
So the best thing here is to introduce some Directory like object as we discussed. I'll introduce this in future.